client/package.json

```diff
@@ -3,8 +3,6 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/user-event": "^7.2.1",
```

Comment:
Why are we getting rid of the testing packages?

Reply:
I had no idea that was being removed will check first thing in the morning !